Pocket/Pocket/Models/TransactionResponse.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TransactionResponse.swift
+//  Pocket
+//
+//  Created by Luis De Leon on 6/5/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public class TransactionResponse {
+    public var network: Network?
+    public var serialized_tx = ""
+    public var tx_metadata = Dictionary<AnyHashable, Any>()
+    public var hash = ""
+    public var metadata = Dictionary<AnyHashable, Any>()
+    public var error = false
+    public var error_msg = ""
+    
+    init(json: JSON) {
+        guard let responseObject = json.dictionaryObject else {
+            return
+        }
+        
+        if responseObject["network"] != nil{
+            network?.tokenID = String(describing: responseObject["network"])
```

Comment:
Shouldn't this be: `network = Network.init(tokenID: responseObject["network"].stringValue)`